src/features/content-explorer/content-explorer-modal-container/ContentExplorerModalContainer.js

```diff
@@ -70,6 +70,10 @@ class ContentExplorerModalContainer extends Component {
         initialFoldersPath: FoldersPathPropType.isRequired,
         /** Initial items that will show up as selected */
         initialSelectedItems: PropTypes.object,
+        /** Items that will show up as selected */
+        autoSelectedItems: PropTypes.object,
```

Comment:
for my understanding: does an item in `autoSelectedItems` only deselect when used with `deselectAutoSelectedItem`?

i.e. what happens if a value is passed for the `autoSelectedItems` prop but no callback is passed for the `deselectAutoSelectedItem` prop?

Reply:
No, autoSelectedItems are not disabled. That means that user can deselect an autoSelectedItem. deselectAutoSelectedItem prop is used to clear autoSelectedItem in parent component when methods like `deselectAll` are called in contentExplorer